### PR TITLE
Rel v4.13.0.5

### PR DESCRIPTION
**Pull request by Sakoram**

### CODE REVIEW

File: _posts/2021-03-04-PC-Engines-Firmware-v4-13-0-5.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.13.0.5
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** e7a68ec.
+2. **Added** resource allocator v4 support. It supports multiple ranges for
```
Comment (miczyg1) [inline _posts/2021-03-04-PC-Engines-Firmware-v4-13-0-5.md:15]: PR is not merged https://github.com/pcengines/coreboot/pull/465
Please remove

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: _posts/2021-03-04-PC-Engines-Firmware-v4-13-0-5.md (hunk 1 above)
Comment (Sakoram) [reply _posts/2021-03-04-PC-Engines-Firmware-v4-13-0-5.md]: Removed.

File: _posts/2021-04-13-PC-Engines-Firmware-v4-13-0-5.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+![Mainline test results](https://cloud.3mdeb.com/index.php/s/sF67YzfFekgYn8d/preview)
+
+* Mainline:
+  * PASSED: **569** (+1)
+  * FAILED: **4**
+  * PASSED [%]: **99.30**
+
+### Key Changes in testing
+
+1. There were no major changes in the testing infrastructure.
```
Comment (miczyg1) [inline _posts/2021-04-13-PC-Engines-Firmware-v4-13-0-5.md:50]: Where is the update about apu6 testing?  @StanislawAniola

**Review by artur-rs — APPROVED** [review]

File: _posts/2021-04-13-PC-Engines-Firmware-v4-13-0-5.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.13.0.5"
+date:   2021-04-13
```
Comment (miczyg1) [inline _posts/2021-04-13-PC-Engines-Firmware-v4-13-0-5.md:4]:
```suggestion
date:   2021-04-19
```

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**